src/logics/LRALogic.cc

```diff
@@ -161,4 +163,58 @@ PTRef LRALogic::insertTerm(SymRef sym, vec<PTRef> &&terms) {
     if (sym == get_sym_Real_DIV())
         return mkRealDiv(terms);
     return LALogic::insertTerm(sym, std::move(terms));
+}
+
+/**
+ * Normalizes a sum term a1x1 + a2xn + ... + anxn + c such that the leading coefficient is either 1 or -1.
+ * Additionally, the normalized term is separated to constant and non-constant part, and the constant is modified as if
+ * it was placed on the other side of an equality.
+ *
+ * @param sum
+ * @return Constant part of the normalized sum as LHS and non-constant part of the normalized sum as RHS
```

Comment:
You are right. I think I originally wanted to have a pair of PTRefs and then realized than in LIA I need the Rational value rather than PTRef. It seems to be fine to unify it in the way you suggest.